src/adagrams.js

```diff
@@ -1,15 +1,147 @@
+/**
+ * The function drawLetters will populate a letter bank allLetters
+ * that contains the letter and n frequency from LETTER_POOL.
+ * Then, the function will generate 10 random letters and store
+ * them in a listOfstrings
+ */
 export const drawLetters = () => {
-  // Implement this method for wave 1
+  const LETTER_POOL = {
+    A: 9,
+    B: 2,
+    C: 2,
+    D: 4,
+    E: 12,
+    F: 2,
+    G: 3,
+    H: 2,
+    I: 9,
+    J: 1,
+    K: 1,
+    L: 4,
+    M: 2,
+    N: 6,
+    O: 8,
+    P: 2,
+    Q: 1,
+    R: 6,
+    S: 4,
+    T: 6,
+    U: 4,
+    V: 2,
+    W: 2,
+    X: 1,
+    Y: 2,
+    Z: 1,
+  };
+  const allLetters = [];
+  for (const element in LETTER_POOL) {
+    const freqLetter = LETTER_POOL[element];
+
+    for (let step = 0; step < freqLetter; step++) {
+      allLetters.push(element);
+    }
+  }
+
+  const listOfstrings = [];
```

Comment:
Since these aren't just strings but specifically letters so putting that in the name would make things clearer:

```suggestion
  const listOfLetters = [];
```